docs/pages/modules/appdelegate-subscribers.md

```diff
@@ -0,0 +1,81 @@
+---
+title: AppDelegate Subscribers
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+Some native modules need the user to manually apply changes in the **AppDelegate.m** file (e.g. app state changes, linking, notifications) to make them working properly. To automate this process, Expo Modules provide a mechanism that allows your library to capture calls to `AppDelegate` functions. The only thing that the user needs to do to use your library is to make `AppDelegate` class inherit from `ExpoAppDelegate`.
```

Comment:
Need to add some info about how if users implement AppDelegate methods in their project AppDelegate, they'll also need to add `[super ...]` for the method otherwise the subscriber methods won't be invoked.